support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -247,10 +282,6 @@ export function ThreeTierLanding(): JSX.Element {
 		(state) => state.common.internationalisation,
 	);
 
-	const tierCards = abParticipations.additionalBenefits
-		? tierCardsTote
-		: tierCardsNoTote;
```

Comment:
We'll have to add the totes benefits (which will be book benefits) in a different way in future.

it'll be a good test for when handing over to Value & Growth to ensure it's easy to do.